app/src/lib/git/checkout-index.ts

```diff
@@ -26,12 +26,15 @@ export async function checkoutIndex(
     return
   }
 
+  const options = {
+    successExitCodes: new Set([0, 1]),
```

Comment:
Required because of: https://github.com/git/git/commit/7e41061588eab25d0c015e79aa8d04956bee482a#diff-01d6b62670b6aa301c5b409a12bef7d46a8de5501c06d2b3d1d4609d53b86f17R264-R265